sdk/healthinsights/azure-health-insights-radiologyinsights/src/main/resources/META-INF/azure-health-insights-radiologyinsights_apiview_properties.json

```diff
@@ -20,7 +20,6 @@
     "com.azure.health.insights.radiologyinsights.models.DocumentContent": "AzureHealthInsights.DocumentContent",
     "com.azure.health.insights.radiologyinsights.models.DocumentContentSourceType": "AzureHealthInsights.DocumentContentSourceType",
     "com.azure.health.insights.radiologyinsights.models.DocumentType": "AzureHealthInsights.DocumentType",
-    "com.azure.health.insights.radiologyinsights.models.Encounter": "AzureHealthInsights.Encounter",
```

Comment:
There is some removal of model classes.

Please delete the `com/azure/health/insights/radiologyinsights/models` and regen the module, to remove any class that not included in this GA.

Reply:
I regenerated the module as requested and the Encounter class is gone